My work seccion

Sirvelasque opened the pull request:
Here I put my work section with the space and samples where my projects will be show with a picture, summary and tools info, also I added a buton for more details and replicated 3 times in a 4 items 1 column grid

Sirvelasque committed:
add container, add image container and text needed

create the works container and samples with numbers except for the first one which contain the first content

Sirvelasque committed:
Update style.css

Sirvelasque committed:
display container as grid and add the section header

add the title and a divider to then display the container as flex and specify the amount of items, width and heigh of them

Sirvelasque committed:
define project image width and heigh, and suport text font

define the font size and family of the sup text

Sirvelasque committed:
style language tags for the projects

n/a

Sirvelasque committed:
make and style 'see project' button

add a shadow on hover for the button and change color when is pressed

Sirvelasque committed:
add style on disable button and copy the work sample grid item

change the color of buttons on disabled and copy the first item in the works container to paste it in the other containers as a sample to work on

Sirvelasque committed:
background setted and z-index change

move the middle_cicles container in the z-index so it doesn't-t interfere with the buton

Sirvelasque committed:
linter errors

fixed white space errors

Sirvelasque committed:
linter errors fix

Sirvelasque committed:
space linter error

rotimiazeez commented on html/index.html:
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
```
- [ ] Please note that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` 😉

rotimiazeez commented on html/index.html:
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
+            <button type="button" class="project_btn">See project</button>
+        </div>
+      </div>
+    </section>
+
```
- [ ] Kindly make sure the about section is added to complete this milestone👍

Sirvelasque replied:
hello @rotimiazeez ! thank you for the review, can I have one more hint? I mean, the template doesn't show an about section so I would like to have an idea about how to solve this

mateo951 replied:
@Sirvelasque yes you are right 👀 There is no about me section on your template 🤯

![image](https://user-images.githubusercontent.com/15807343/149606923-8477201c-4023-477f-a2d5-b76e07f37723.png)

rotimiazeez reviewed (changes requested):
# Hi @Sirvelasque 👍, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
change the tag for work tools

the list was tagged by divs now they are lister as a ul with respective li items

Sirvelasque committed:
linter errors solved

there was an unexpected div

mateo951 commented on html/index.html:
```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul class="work_tools_tags">
+            <li class="tool_tag">css</li>
+            <li class="tool_tag">html</li>
+            <li class="tool_tag">bootstrap</li>
+            <li class="tool_tag">Ruby</li>
+          </ul>
```
Addressed 👌🔥

mateo951 reviewed (approved):
Hi @Sirvelasque, 

Great job making the adjustments required by your previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
